ACE/ace/os_include/net/os_if.h

```diff
@@ -102,7 +102,8 @@ struct  ifconf {
 # define IFF_BROADCAST 0x2
 #endif /* IFF_BROADCAST */
 
-#if defined ACE_HAS_WINSOCK2 && defined ACE_LACKS_IF_NAMEINDEX
+#if (defined ACE_LACKS_IF_NAMEINDEX && defined ACE_HAS_WINSOCK2) || \
+  defined ACE_LACKS_STRUCT_IF_NAMEINDEX
```

Comment:
Can't we add ACE_LACKS_STRUCT_IF_NAMEINDEX to config-win32-common.h, that makes this file again simpler.

Reply:
ACE_HAS_WINSOCK2 is always defined on Windows, so very likely ACE_LACKS_STRUCT_IF_NAMEINDEX can just be added to config-win32-common.h